platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -443,39 +464,38 @@ check_kernel_flags() {
 # See https://github.com/moby/moby/blob/master/contrib/check-config.sh
 #Reference Issue : https://github.com/Azure/iotedge/issues/5812
 check_cgroup_heirachy() {
-    wrap_debug "Checking cgroup hierarchy..."
+    wrap_debug_message "Checking cgroup hierarchy..."
     EXITCODE=0
     if [ "$(stat -f -c %t /sys/fs/cgroup 2>/dev/null)" = '63677270' ]; then
         wrap_good 'cgroup hierarchy' 'cgroupv2'
         cgroupv2ControllerFile='/sys/fs/cgroup/cgroup.controllers'
         if [ -f "$cgroupv2ControllerFile" ]; then
-            echo '  Controllers:'
             for controller in cpu cpuset io memory pids; do
                 if grep -qE '(^| )'"$controller"'($| )' "$cgroupv2ControllerFile"; then
-                    echo "  - $(wrap_good "$controller" 'available')"
+                    wrap_good "$controller" 'available'
                 else
-                    echo "  - $(wrap_bad "$controller" 'missing')"
+                    wrap_bad "$controller missing"
                     EXITCODE=1
                 fi
             done
         else
-            wrap_bad "$cgroupv2ControllerFile" 'nonexistent??'
+            wrap_bad "$cgroupv2ControllerFile nonexistent??"
             EXITCODE=1
         fi
         # TODO find an efficient way to check if cgroup.freeze exists in subdir
     else
         cgroupSubsystemDir="$(sed -rne '/^[^ ]+ ([^ ]+) cgroup ([^ ]*,)?(cpu|cpuacct|cpuset|devices|freezer|memory)[, ].*$/ { s//\1/p; q }' /proc/mounts)"
         cgroupDir="$(dirname "$cgroupSubsystemDir")"
         if [ -d "$cgroupDir/cpu" ] || [ -d "$cgroupDir/cpuacct" ] || [ -d "$cgroupDir/cpuset" ] || [ -d "$cgroupDir/devices" ] || [ -d "$cgroupDir/freezer" ] || [ -d "$cgroupDir/memory" ]; then
-            echo "$(wrap_good 'cgroup hierarchy' 'properly mounted') [$cgroupDir]"
+            wrap_good "cgroup hierarchy at $cgroupDir" "properly mounted"
```

Comment:
Please check the permissions of the path & these directories as well plx.

Reply:
Not needed. See comment above.